e2e/config/vhd.go

```diff
@@ -72,7 +72,7 @@ var (
 	}
 
 	// without kubelet, kubectl, credential-provider and wasm
-	VHDUbuntu2204Gen2ContainerdAirgapped = &Image{
+	VHDUbuntu2204Gen2ContainerdAirgappedPinnedVersion = &Image{
```

Comment:
nit: `VHDUbuntu2204Gen2ContainerdAirgappedK8sNotCached` to align with the corresponding scenario name?